src/PooledArrays.jl

```diff
@@ -428,6 +428,8 @@ function Base.vcat(a::PooledArray{T, <:Integer, 1}, b::PooledArray{S, <:Integer,
     return PooledArray(RefArray(newrefs), convert(Dict{U, refT}, newlabels))
 end
 
+fast_sortable(y::PooledArray{<:Integer}) = y
```

Comment:
@piever Should we additionally check `isbitstype(eltype(y))` and otherwise use the "slow" path?

Reply:
Good point, I imagine non-isbits comparison can be quite slow, even though in practice I doubt anyone has a `PooledArray{BigInt}` or things like that. I've changed that to the slow path. The important part (for me) is that `fast_sortable` is idempotent which should always be the case as `sortperm` returns isbits integers.